### PR TITLE
Expose http trace data via PM API

### PR DESCRIPTION
**Pull request by kraman**
Connect to strongloop/strong-mesh-models#12

### CODE REVIEW

File: lib/server.js [hunk 1]
```diff
@@ -294,6 +294,50 @@ Server.prototype._onMetrics = function _onMetrics(req, callback) {
...
+  var q = {where: {serviceInstanceId: 1, pid: +req.processId, stopTime: null}};
+  Process.findOne(q, function(err, proc) {
+    assert.ifError(err);
+    if (!proc) {
+      debug('Ignoring trace for unknown process: %d', req.processId);
+      return callback();
+    }
+
+    async.each(req.trace.list,
+      saveTrace.bind(this, proc),
```
Comment (sam-github) [inline lib/server.js]: Shouldn't `this` be null? It's not defined to anything at this point, and not used by saveTrace().

**Comment by sam-github** [comment]
One comment, but other than that, LGTM.

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Pull request merged**